src/pages/Governance/VotingWallet/index.tsx

```diff
@@ -134,81 +161,86 @@ export const VotingWalletUi: React.FC<VotingWalletUiProps> = ({
           </PrimaryButton>
         )}
 
-        {connectedWallet && !userHasLockedXVS && (
+        {showDepositXvs && (
           <ButtonWrapper className="text-offWhite hover:no-underline sm:w-auto lg:w-full" asChild>
             <Link to={routes.vaults.path}>{t('vote.depositXvs')}</Link>
           </ButtonWrapper>
         )}
 
-        {connectedWallet && userHasLockedXVS && (
+        {showDelegateButton && (
           <PrimaryButton
             className="text-offWhite sm:w-auto lg:w-full"
             onClick={() => setDelegateModelIsOpen(true)}
+            data-testid={TEST_IDS.delegateButton}
           >
             {previouslyDelegated ? t('vote.redelegate') : t('vote.delegate')}
           </PrimaryButton>
         )}
-      </Paper>
+      </Card>
 
-      <Paper css={[styles.votingWalletPaper, styles.voteSection]}>
-        <Typography variant="body2" color="textPrimary" css={styles.toVote}>
-          {t('vote.toVoteYouShould')}
-        </Typography>
+      {voteProposalFeatureEnabled && (
+        <>
+          <Card className="mt-6 flex flex-col items-start justify-start p-6 lg:justify-between">
+            <p className="mb-4 font-semibold leading-normal tracking-[0.3px]" color="textPrimary">
```

Comment:
No need for the  `leading-normal` and `tracking-[0.3px]` classes. The `color` prop should also be removed as it's not relevant to a `p` tag.

Reply:
Same as https://github.com/VenusProtocol/venus-protocol-interface/pull/1781#discussion_r1403217341